docs/reference/ml/df-analytics/apis/delete-trained-model.asciidoc

```diff
@@ -0,0 +1,59 @@
+[role="xpack"]
+[testenv="platinum"]
+[[delete-inference]]
+=== Delete trained model API
+[subs="attributes"]
+++++
+<titleabbrev>Delete trained model</titleabbrev>
+++++
+
+Deletes an existing trained {dfanalytics} model that is currently not referenced 
```

Comment:
These models in the future do not have to be `dfanalyics` models. They just happen to be that right now. I don't think we want to restrict them in documentation right now.

I am not sure if we want to call them `trained inference model`? I do agree that we need some color around `trained model` so folks don't confuse them with anomaly detection models.